scripts/setup.sh

```diff
@@ -6,17 +6,28 @@
 #
 # Commands that may fail have "|| return" to avoid continuing or interfering with terminal.
 
-if [ ! -z $NVM_DIR ]; then # skip nvm steps if not available
-  echo 'Configuring node 12...'
-  source ~/.nvm/nvm.sh
-  nvm install || return
+# Install yarn if it does not exist.
+if ! yarn versions &> /dev/null; then
+  echo 'yarn is required for setup, installing...'
+  if ! brew --version &> /dev/null; then
+    echo 'brew is required to install yarn, see https://docs.brew.sh/Installation'
+    exit 0
+  fi
+  brew install yarn
+fi
+
+if ! memcached --version &> /dev/null; then
+  brew install memcached
```

Comment:
The `brew bundle` command is supposed to work similarly (only install what isn't already installed). Are you testing for each dependency in case developers have them installed other [non-brew] ways?

Reply:
> developers have them installed other [non-brew] ways
Yup, exactly. I've run into a lot of trouble in the past with multiple installs so its nice to check ahead of time.